src/PlutoTeachingTools.jl

```diff
@@ -1,5 +1,6 @@
 module PlutoTeachingTools
 
+using PlutoUI.ExperimentalLayout: Div
```

Comment:
Is there a reason to use `using` here instead of `import`?

Reply:
you should only use `import` if you extend a function.

(This consistent with the rest of the package.)